crates/pop-cli/src/commands/up/parachain.rs

```diff
@@ -151,6 +159,26 @@ impl ZombienetCommand {
 	}
 }
 
+pub(crate) async fn run_custom_command(
+	spinner: &ProgressBar,
+	command: &str,
+) -> Result<(), anyhow::Error> {
+	spinner.set_message(format!("Running command: {}", command.to_string()));
+	sleep(Duration::from_secs(15)).await;
+
+	// Split the command into the base command and arguments
+	let mut parts = command.split_whitespace();
```

Comment:
Instead of parsing here, we can use `clap`, and has has the arguments like this:
```
/// The command to run after the network has been launched.
#[clap(name = "cmd", short = 'c', long)]
command: Option<String>,
/// The script arguments, encoded as strings.
#[clap(long, num_args = 0..)]
args: Vec<String>,
```

Reply:
Does this not require the specification of an additional `--args`? We dont want this feature to make the usage more complex, hence my suggestion of having it as a single arg.

Reply:
Yes it needs an additional `--args`. At first it looked weird to me everything in one command, but I see your point.

Example of run a script `fund-dev-accounts` that has an argument for the balance to fund:
```
--cmd "./fund-dev-accounts 1000" 
```
or
```
--setup_script fund-dev-accounts --args 1000
```

We can start with `--cmd "./fund-dev-accounts 1000"` and iterate with user's feedback.

Reply:
I agree that the args is probably easier for us, but I think it probably clutters things with noise. A single arg keeps the feature self-contained.

I like starting with simple and taking feedback to improve.